README.md

```diff
@@ -5,7 +5,7 @@
 
 # What?
 
-This is a docker container intended to run inside a kubernetes cluster to collect config maps with a specified label and store the included files in an local folder. It can also send an HTTP request to a specified URL after a configmap change. The main target is to be run as a sidecar container to supply an application with information from the cluster. The contained Python script is working with the Kubernetes API 1.10
+This is a docker container intended to run inside a kubernetes cluster to collect config maps with a specified label and store the included files in an local folder. It can also send an HTTP request to a specified URL after a configmap change. The main target is to be run as a sidecar container to supply an application with information from the cluster. The contained Python script is working with the Kubernetes API 1.16.
```

Comment:
please can you adjust the k8s version, that it become clear to users that the script will also work with lower k8s version?

Reply:
Minimum version can be found here:

https://github.com/kiwigrid/k8s-sidecar/blob/master/requirements.txt

Reply:
Yep, that's much better!

Reply:
Change pushed.